<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="original">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>ORIGINAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_13">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_14">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_15">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>120</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>1-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_25">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_22">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_23">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_24">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_21">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>2-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_35">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_32">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_33">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_44">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_41">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_34">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_43">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_42">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_31">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>3-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_45">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>480</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>4-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_53">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-3</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_51">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_52">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_55">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="aug_54">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>600</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>5-4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIZE = 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>290</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIZE = 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIZE = 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>530</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIZE = 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>650</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SIZE = 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y11111</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y22222</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y33333</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>96</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y44444</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y55555</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>251</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sliderLayout"/>
   </widget>
   <widget class="QPushButton" name="regenMatrix">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(158, 90, 214)</string>
    </property>
    <property name="text">
     <string>Regenerate Matrix</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
